src/dev-server.js

```diff
@@ -0,0 +1,89 @@
+import 'babel-polyfill';
+import {DockerExecServer,DockerExecClient} from 'docker-exec-websocket-server';
+import Docker from 'dockerode-promise';
+import express from 'express';
+import http from 'http';
+import fs from 'fs';
+import slugid from 'slugid';
+import path from 'path';
+import reactViews from 'express-react-views'
+
+var debug = require('debug')('dev-server:');
+
+const PORT = 34678;
+const DOCKER_SOCKET = '/var/run/docker.sock'
+const PATH = '/'+slugid.v4();
+
+var dockerServer, container, docker,server,dockerClient;
+//check if docker is running i.e socket is available
+var start = async ()=>{
+  debug('Process started');
+  if(!fs.statSync(DOCKER_SOCKET).isSocket()){
+    debug("Docker is not running");
+    throw new Error("Docker is not running");
+  }
+  //Set up the docker server
+  docker = new Docker({socketPath:DOCKER_SOCKET});
+  await docker.pull('ubuntu');
+  container = await docker.createContainer({
+    Image:'gdb-test',
+    cmd: ['sleep','600']
+  });
+
+  await container.start();
+  debug("Container started");
+  server = http.createServer();
+  server.listen(PORT);
+  debug("Server listening...")
+  dockerServer = new DockerExecServer({
+    server: server,
+    containerId: container.id,
+    path: PATH
+  });
+  debug("Docker server created");
+
+  //set up express
+  var app = express();
+  app.set('views',__dirname+'client');
+  app.set('view engine','jsx');
+  app.engine('jsx',reactViews.createEngine());
```

Comment:
Let's not render jsx on the server... Can we just put a simple dump HTML file into the `client/` folder..

Something like:
```html
<html>
<script src="....something that webpack built">
<body>...
</html>
```